src/index.js

```diff
@@ -381,28 +390,36 @@ class JanusAdapter {
     };
   }
 
-  configureSubscriberSdp(originalSdp) {
+  configurePublisherSdp(jsep) {
+    jsep.sdp = jsep.sdp.replace(/a=fmtp:(109|111).*\r\n/g, (line, pt) => {
```

Comment:
is the carriage return part of the spec?

Reply:
(if not may want to just match `$`)

Reply:
Yes, although on rereading it also implies that it's polite to tolerate bare newlines, so I might fix it up.

Reply:
It's more awkward to write the regex to deal with either, and I don't think it's more useful or clearer (e.g. `sdp.writeFmtp` always emits CRLF, so it would be pretty weird anyway to match LF and then replace it with CRLF.) So I'm leaving it be for now.